holidays/countries/__init__.py

```
@@ -62,6 +62,11 @@
 from holidays.countries.curacao import Curacao, CW, CUW
 from holidays.countries.cyprus import Cyprus, CY, CYP
 from holidays.countries.czechia import Czechia, CZ, CZE
+from holidays.countries.democratic_republic_of_the_congo import (
```

Flake8 (7.2.0)

[error] 'holidays.countries.democratic_republic_of_the_congo.DemocraticRepublicOfTheCongo' imported but unused

(F401)